modules/ept-threads/migrations/20170201215933-add-multiple-owners.js

```diff
@@ -0,0 +1,27 @@
+var dbm = global.dbm || require('db-migrate');
+var fs = require('fs');
+var path = require('path');
+
+exports.up = function(db, callback) {
+  var filePath = path.join(__dirname + '/sqls/20170201215933-add-co-users-up.sql');
```

Comment:
This refers to the sql file as add-co-users-up.sql when the filename is actually add-co-owners-up.sql

Same issue with down file below.

We currently dont support putting migrations in the modules themselves and they are typically written directly into core-pg. We hope to fix that in the future, but for now this shouldn't interfere with us testing